### PR TITLE
add tests for dropdown logs

### PR DESCRIPTION
**Pull request by karuppiah7890**
this adds tests for issues like https://github.com/getgauge/taiko/issues/726

### CODE REVIEW

**Comment by karuppiah7890** [comment]
@nehashri I agree there shouldn't be a test for every scenario. I saw three scenarios to test

1. Display exists
2. Display selected index
3. Display selected value

I updated the PR based on the same. And nice catch on the tests not waiting for event to occur, I fixed that too. I'll raise a PR for

File: test/unit-tests/dropDown.test.js [hunk 1]
```diff
@@ -60,11 +86,31 @@ describe(test_name, () => {
...
         it('test exists()', async () => {
             expect(await dropDown('dropDownWithWrappedInLabel').exists()).to.be.true;
             expect(await dropDown('dropDownWithWrappedInLabel').value()).to.not.equal('mercedes');
         });
     });
+
+    describe('test logs for dropdown', () => {
+        it('should show exists', async () => {
+            let validatePromise =  validateEmitterEvent('success', 'Exists');
+            expect(await dropDown('Cars').exists()).to.be.true;
```
Comment (nehashri) [inline test/unit-tests/dropDown.test.js]: There is no need for expect here.

Comment (karuppiah7890) [reply test/unit-tests/dropDown.test.js]: Oops. Yes. Fixed it 👍

**Comment by karuppiah7890** [comment]
Updated the PR

**Comment by karuppiah7890** [comment]
@nehashri Tests have passed now, anything else to do to merge?

**Pull request merged**